scripts/lib/CIME/XML/env_batch.py

```diff
@@ -591,21 +608,31 @@ def get_queue_specs(self, queue):
         return None
 
     def get_default_queue(self):
-        node = self.get_optional_child("queue", attributes={"default" : "true"})
-        if node is None:
-            node = self.get_optional_child("queue")
+        bs_nodes = self.get_children("batch_system")
+        for bsnode in bs_nodes:
+            qnodes = self.get_children("queues", root=bsnode)
+            for qnode in qnodes:
+                node = self.get_optional_child("queue", attributes={"default" : "true"}, root=qnode)
+                if node is None:
+                    node = self.get_optional_child("queue", root=qnode)
         expect(node is not None, "No queues found")
         return node
```

Comment:
Need an initialization above: e.g. `node = None`.

Reply:
@amametjanov , done (i pushed directly to master).